config/README.md

```diff
@@ -381,6 +381,17 @@ test_extras = """
 """
 ```
 
+If your package uses [`mxdev`](https://pypi.org/project/mxdev/) to handle source checkouts for dependencies, you can set the `use_mxdev` key to ensure `tox` will first run mxdev.
+
+You also will need to manually set the installation of additional packages being pulled by `mxdev` in the `test_deps_additional` key.
+
+```toml
+[tox]
+use_mxdev = true
+constrain_package_deps = false
+test_deps_additional = "-esources/plonegovbr.portal_base[test]"
```

Comment:
Minor formatting issue:
```suggestion
test_deps_additional = """
    -esources/plonegovbr.portal_base[test]
"""
```

I guess we want to have multiple external repositories and they have to go one per line 🍀

Reply:
Another question on this, and again me being noob on `mxdev` 🤓 :

`mxdev` provides a way to generate that list of _in development_ packages, so they can be automatically filled in? that would make maintenance of the changes easier on `plone/meta` side, as you would not have to keep calling `plone/meta` every time you want something in/out of development mode, but rather you would only need to run the `mxdev` commands to add a package in/out of it. ✨

Reply:
About `mxdev`, it generates a `requirements-mxdev.txt` file where the list of development sources is available under the `# mxdev development sources`:
```
###############################################################################
# mxdev development sources
-e ./sources/plonegovbr.portal_base[test] --pre


```
But I decided not to be too clever and try to extract the list from there. I would suggest a new feature on `mxdev` to  return only those lines, if possible